onnx/test/version_converter_test.py

```diff
@@ -2184,6 +2184,90 @@ def test(input_shape, scale_shape, axis, block_size) -> None:
         with context_manager:
             test(y_shape, scale_shape, axis, block_size)
 
+    def test_external_data_version_conversion(self) -> None:
+        # Create a model with external data
+        shape = (200, 300)
+        random_data = np.random.rand(*shape).astype(np.float32)
+
+        initializer_tensor = onnx.helper.make_tensor(
+            name="initializer_tensor",
```

Comment:
Is this expected to be external too?

Reply:
What do you mean? This is the tensor that's supposed be stored in external data, which is what the checks are checking later on.

Reply:
Sorry I meant `initializer_scalar`

Reply:
`initializer_scalar` is not in external data from local testing, possibly because we didn't supply raw data. But it should be ok, as the point of the test is to make sure existing external data is properly preserved.